system/cc-ceph/templates/cephobjectstore.yaml

```diff
@@ -47,6 +47,13 @@ spec:
   {{- end }}
     annotations:
       checksum/config: {{ include (print $.Template.BasePath "/ceph-config-override.yaml") . | sha256sum }}
```

Comment:
it also makes sense to update the deployment's `checksum/config`, if the sidecar is enabled and the configmap was updated.